mediator/med_internalstate_mod.F90

```diff
@@ -30,7 +30,8 @@ module med_internalstate_mod
   integer, public :: compice = 5
   integer, public :: comprof = 6
   integer, public :: compwav = 7
-  integer, public :: ncomps =  7 ! this will be incremented if the size of compglc is > 0
+  integer, public :: compdat = 8
```

Comment:
@billsacks do we have (or need) a multi glacier test to make sure this doesn't break that feature?

Reply:
Thanks for checking. Yes, we have two multi-glacier tests in the prealpha test list:
SMS_D_Ld5_P24x1.f10_f10_ais8gris4_mg37.I1850Clm50SpGag.izumi_nag.cism-test_coupling
MULTINOAIS_Ly2.f10_f10_ais8gris4_mg37.I1850Clm50SpRsGag.cheyenne_intel.cism-change_params